apps/web/src/components/modal-with-close-button.tsx

```diff
@@ -90,20 +91,27 @@ export function ModalWithCloseButton({
           <Dialog.Content
             className={cn('serlo-modal', className)}
             data-modal-state={isOpen ? 'open' : 'closed'}
+            aria-describedby={undefined}
           >
             {title ? (
               <Dialog.Title className={cn('serlo-h2', extraTitleClassName)}>
                 {title}
               </Dialog.Title>
-            ) : null}
+            ) : (
+              <VisuallyHidden.Root asChild>
```

Comment:
This was added as there was a console warning that Dialog.Title is necessary

Reply:
The console error:

>`DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.

> If you want to hide the `DialogTitle`, you can wrap it with our VisuallyHidden component.

I don't think that we should just silence the error by adding an empty `VisuallyHidden`. If we're addressing the error, let's actually help the screen reader users by adding a helpful title for them?